content/geometry/object-human/human-model-structure.adoc

```diff
@@ -2,1213 +2,997 @@
-        |_ [[../geometry/object-classes.html#_standing_point_left_t Standing_Point_Left (T)]]
-|_ [[../geometry/object-classes.html#_body Body]]
-|_ [[../geometry/object-classes.html#_clothing Clothing]]
-|_ [[../geometry/object-classes.html#_hair Hair]]
-end legend
+|Grp_Root
+  |Skeleton
+    |Root
+      |Lower_Spine
+        |Hip_Left
+        |Hip_Right
+        |Middle_Spine
+          |Upper_Spine
+            |Neck
+              |Head
+                |Eye_Left
+                |Eye_Right
+          |Shoulder_Left
+            |Upper_Arm_Left
+              |Lower_Arm_Left
+                |Hand_Left
+                  |Full_Fingers_Left
+                  |Full_Thumb_Left
+            |Upper_Arm_Right
+              |Lower_Arm_Right
+                |Hand_Right
+                  |Full_Fingers_Right
+                  |Full_Thumb_Right
+      |Upper_Leg_Left
+        |Lower_Leg_Left
+          |Foot_Left
+            |Full_Toes_Left
+      |Upper_Leg_Right
+        |Lower_Leg_Right
+          |Foot_Right
+            |Full_Toes_Right
+  |Grp_Accessories
+    |Accessorie
+  |Grp_Body
+    |Human
+  |Grp_Clothing
+    |Vest
+  |Grp_Hair
+    |Hair
 ----
 
-== Root_Bone (T)
 
-Add a description
+== Grp_Root 
 
-.<Table title>
-[%header, cols="20, 80"]
-|===
-
-2+^| <Header>
-
-| *Origin*
-| ...
-
-| *x-axis*
-| ...
-
-| *y-axis*
-| ...
-
-| *z-axis*
-| ...
-|===
-
-
-Add a figure.
-
-.<Figure caption>
-//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
-
-== Hip (T)
-
-Add a description
-
-.<Table title>
-[%header, cols="20, 80"]
-|===
-
-2+^| <Header>
-
-| *Origin*
-| ...
-
-| *x-axis*
-| ...
-
-| *y-axis*
-| ...
-
-| *z-axis*
-| ...
-|===
-
-
-Add a figure.
-
-.<Figure caption>
-//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
-
-== Lower_Spine (T)
-
-Add a description
-
-.<Table title>
-[%header, cols="20, 80"]
-|===
-
-2+^| <Header>
-
-| *Origin*
-| ...
-
-| *x-axis*
-| ...
-
-| *y-axis*
-| ...
-
-| *z-axis*
-| ...
-|===
-
-
-Add a figure.
-
-.<Figure caption>
-//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
-
-== Upper_Spine (T)
-
-Add a description
-
-.<Table title>
-[%header, cols="20, 80"]
-|===
-
-2+^| <Header>
-
-| *Origin*
-| ...
-
-| *x-axis*
-| ...
-
-| *y-axis*
-| ...
-
-| *z-axis*
-| ...
-|===
-
-
-Add a figure.
-
-.<Figure caption>
-//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
-
-== Neck (T)
-
-Add a description
-
-.<Table title>
-[%header, cols="20, 80"]
-|===
-
-2+^| <Header>
-
-| *Origin*
-| ...
-
-| *x-axis*
-| ...
-
-| *y-axis*
-| ...
-
-| *z-axis*
-| ...
-|===
-
-
-Add a figure.
-
-.<Figure caption>
-//image::images/Vehicle_Structure_Door_Coord_Frame.svg[width=70%, scalewidth=10cm]
+This group is used as a parent for all following nodes. It can be used to place the whole asset and select the complete node hierarchy at once.
+The origin of the Root group is set at the world origin.
```

Comment:
I would not call it "world origin" as it is the origin of the object, i.e. center of bounding box on the ground, correct?
Same for the other Grp nodes.

Reply:
I would say if it is done right, the world origin should equal the origin of the object (middle of the bounding box on the ground), so that the transforms are set to 0 0 0. But I can change it here or state, that those should be the same in the end, if it is set up correctly.

Reply:
Good point. I will change it.